roles/sap_vm_provision/tasks/platform_ansible/gcp_ce_vm/execute_main.yml

```diff
@@ -112,25 +139,6 @@
         auth_kind: "serviceaccount"
         service_account_file: "{{ sap_vm_provision_gcp_credentials_json }}"
 
-    - name: Gather GCP VPC Subnet information
-      no_log: "{{ __sap_vm_provision_no_log }}"
-      register: __sap_vm_provision_task_gcp_vpc_subnet_info
-      google.cloud.gcp_compute_subnetwork_info:
-        project: "{{ sap_vm_provision_gcp_project }}"
-        region: "{{ sap_vm_provision_gcp_region }}"
-        filters:
-          - name = {{ sap_vm_provision_gcp_vpc_subnet_name }}
-        auth_kind: "serviceaccount"
-        service_account_file: "{{ sap_vm_provision_gcp_credentials_json }}"
-
-    - name: Gather GCP Private DNS information
```

Comment:
Removing this Ansible Task will fail the next task `GCP Private DNS Records for hosts`

Reply:
Task is removed because it was twice there, already gathered in:
```yaml
    - name: Identify GCP Subnetwork (VPC Subnet)
      no_log: "{{ __sap_vm_provision_no_log }}"
      register: __sap_vm_provision_task_gcp_vpc_subnet_info
      google.cloud.gcp_compute_subnetwork_info:
        project: "{{ sap_vm_provision_gcp_project }}"
        region: "{{ sap_vm_provision_gcp_region }}"
        filters:
          - name = "{{ sap_vm_provision_gcp_vpc_subnet_name }}"
        auth_kind: "serviceaccount"
        service_account_file: "{{ sap_vm_provision_gcp_credentials_json }}"
```

Reply:
My bad, on second look I can see referenced Task `Gather GCP Private DNS information` was just moved upwards in the PR commit - the git diff just didn't show it because a new Task was added too and I didn't realise :)